### PR TITLE
Construct UI: Homepage Icons

### PR DESCRIPTION
**Pull request by gdjohnson**
## Description
This adds the notification and address-selection menu buttons to the Construct homepage. 

## Motivation and Context
See issue #186. The Construct homepage was empty and lacked necessary informational cues for users.

## How Has This Been Tested?
Verified: The login button shows when logged out; the address-picker and notification buttons show when logged in, in a variety of circumstances. The flex styling holds in a variety of browsers.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by zakhap** [comment]
First thought before QA Testing was to factor out the `PseudoHeader` into its own component and file, same with the scss.

**Comment by gdjohnson** [comment]
@zakhap Good thinking, refactoring now!

File: client/styles/pages/home.scss [hunk 1]
```diff
@@ -16,7 +24,7 @@
             font-weight: 600;
             font-size: 28px;
             line-height: 1.25;
-            margin: 42px 0 0;
+            margin-top: 0;
```
Comment (zakhap) [inline client/styles/pages/home.scss:27]: if line 19 is `margin: 0;` then this `margin-top: 0` is unnecessary.

File: client/styles/pages/home.scss [hunk 2]
```diff
@@ -6,6 +6,14 @@
         position: relative;
         padding: 0 20px 60px;
         width: 100%;
+        .pseudo-header {
```
Comment (zakhap) [inline client/styles/pages/home.scss:9]: I would put this in its own scss file and import it into the `PseudoHeader` component.

File: client/styles/pages/home.scss (hunk 1 above)
Comment (gdjohnson) [reply client/styles/pages/home.scss:27]: ah no, the margin-top is an override!

File: client/styles/pages/home.scss (hunk 2 above)
Comment (gdjohnson) [reply client/styles/pages/home.scss:9]: Good call, gonna separate out

**Comment by gdjohnson** [comment]
Should be ready to go whenever.

**Comment by dillchen** [comment]
This looks good, I guess a user will never have an "account / address" at the top level, so we will always see the AddressSwitcher as a carat.

**Pull request merged**